torch/csrc/distributed/autograd/utils.h

```diff
@@ -16,7 +16,8 @@ namespace autograd {
 TORCH_API void addSendRpcBackward(
     DistAutogradContext& autogradContext,
     const torch::distributed::rpc::AutogradMetadata& autogradMetadata,
-    std::vector<torch::Tensor>& tensors);
+    std::vector<torch::Tensor>& tensors,
+    const rpc::WorkerId& dst = rpc::WorkerId());
```

Comment:
We should always pass in the workerId

Reply:
@pritamdamania87 , in the current implementation of `processRPC` in `request_callback_impl.cpp`, is there any way to figure out the destination worker that the `response` is being sent to? I think I could read it off the `autogradContextId ` of the initial rpc, and then take the leading 16 bits which is the worker Id, but not sure if we want to do that.